vue-components/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "scripts": {
     "serve": "vue-cli-service serve",
-    "build": "vue-cli-service build --target lib --formats commonjs --name wikit-vue-components src/main.ts --report --report-json && tsc --emitDeclarationOnly",
+    "build": "vue-cli-service build --target lib --formats commonjs --name wikit-vue-components src/main.ts --report --report-json && tsc --emitDeclarationOnly && VUE_APP_VUE3COMPAT=true vue-cli-service build --target lib --formats commonjs --name wikit-vue-components-vue3compat --no-clean src/main.ts",
```

Comment:
This command is a bit hefty, this is really non-blocking though. I think in a followup, we should split this into two script entries and combine them with `npm-run-all -s`

Reply:
Hmm, but it can’t be `npm-run-all build:*`, because we already have some `build:` tasks that shouldn’t be part of `build` (as I just noticed after implementing this suggestion locally ^^)

Reply:
#529